antarest/study/service.py

```diff
@@ -2731,26 +2709,48 @@ def delete_file_or_folder(self, study_id: str, path: str, current_user: JWTUser)
             current_user: User that called the endpoint
 
         Raises:
-            FileDeletionNotAllowed: if the path does not comply with the above rules
+            ResourceDeletionNotAllowed: if the path does not comply with the above rules
         """
-        study = self.get_study(study_id)
-        assert_permission(current_user, study, StudyPermissionType.WRITE)
+        args = {"path": path}
+        self._alter_user_folder(study_id, args, current_user, RemoveUserResource, ResourceDeletionNotAllowed)
 
-        url = [item for item in path.split("/") if item]
-        if len(url) < 2 or url[0] != "user":
-            raise FileDeletionNotAllowed(f"the targeted data isn't inside the 'User' folder: {path}")
+    def create_folder(self, study_id: str, path: str, current_user: JWTUser) -> None:
```

Comment:
If only creation in user space is allowed, why not just have a method `create_user_folder` ?